tests/test-setup.js

```diff
@@ -0,0 +1,3 @@
+const dotenv = require("dotenv");
+
+dotenv.config({ path: "../.env.test" });
```

Comment:
The path here is wrong - replace it with `./.env.test`
It is causing an issue with the interaction between your test and dev database